main/src/ca/uwaterloo/flix/api/lsp/provider/completion/Completion.scala

```diff
@@ -157,6 +158,18 @@ sealed trait Completion {
         kind             = CompletionItemKind.Class
       )
 
+    case Completion.AutoImportCompletion(label, name, path, ap, documentation, shouldImport) =>
+      val priority = if (shouldImport) Priority.Low else Priority.High
```

Comment:
I think I would make it 

if (shouldImport) Priority.Lower else Priority.Low

because the importing suggestions should not be high